spec/controllers/divisions_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'spec_helper'
 
 describe DivisionsController, :type => :controller do
+  describe "#index" do
+    before :each do
+      Division.delete_all
```

Comment:
Could you add to this a comment like in `spec/features/viewing_user_stats_spec.rb` to remind us to fix this crappy situation requiring this database clearing, e.g. `# TODO: Remove this hack to delete fixtures`?

I'd love to get around to fixing that one day :)